.travis.yml

```diff
@@ -21,11 +25,7 @@ before_script:
   - docker pull "$image_tag" || true
 script:
   - travis_retry docker build --pull --cache-from "$image_tag" --file "$dockerfile" --tag "$image_tag" .
-  - JUNEBUG_IMAGE="$image_tag" docker-compose up --no-build -d && sleep 5
-  # The simplest possible test - Junebug returns 200/OK when we ask for channels
-  - curl -f --user guest:password http://localhost:8080/jb/channels/
-  - curl -f http://localhost:8080/jb/health
-  - JUNEBUG_IMAGE="$image_tag" docker-compose down
+  - (cd seaworthy; pytest -v test.py)
```

Comment:
You could use `pytest -v seaworthy/test.py` if you want to avoid the `cd`.

I've also found `--durations=20` (for varying values of `20`) to show the N slowest things (tests, setups, teardowns, etc.) in the test run.